VK_App/VK_App/ViewControllers/CustomInteractiveTransition.swift

```diff
@@ -0,0 +1,57 @@
+// CustomInteractiveTransition.swift
+// Copyright © RoadMap. All rights reserved.
+
+import UIKit
+
+/// Интерактивная анимация
+final class CustomInteractiveTransition: UIPercentDrivenInteractiveTransition {
+    // MARK: Constants
+
+    private enum Constants {
+        static let magicValue = 0.33
```

Comment:
```suggestion
        static let magicValue = 0.33
```
magic не называем это магией в константе прочти в группе на сайте